contrib/sparse-checkout/update-boost.sh

```diff
@@ -50,7 +50,6 @@ echo '/boost/smart_ptr/*' >> $FILES_TO_CHECKOUT
 echo '/boost/type_index/*' >> $FILES_TO_CHECKOUT
 echo '/boost/type_traits/*' >> $FILES_TO_CHECKOUT
 echo '/boost/system/*' >> $FILES_TO_CHECKOUT
-echo '/boost/tti/*' >> $FILES_TO_CHECKOUT
```

Comment:
This list is contains extraneous lines, at least boost/filesystem and boost/uuid can be removed.
I think that there are tiny bits of optimization. still remain.

Reply:
UPD: turns out that this should include indirect dependencies too.
I have reverted these changes, as boost/graph still depends on boost/tti